README.md

```diff
@@ -78,6 +78,8 @@ Logging of 404 requests is buffered to shield your application from Denial of Se
 
 **Important!** Even if the threshold is set low, you can still receive a lot of requests in the 404 log. In the Admin view (follow "Administer" link in gadget) you can delete suggestions (logged 404 requests). You can find all the logged items in the `BVN.NotFoundRequests` table in your CMS database if you want to manually clear the logged requests (this will not remove any redirects).
 
+**Wildcard!**  If you want to redirect many addresses below a specific one to one new URL, set this to true. If we get a wild card match on this URL, the new URL will be used in its raw format and the old URL will not be appended to the new one.
```

Comment:
This is the wrong place for this documentation. Create a separate section for how to use 404 handler. Maybe call it "Usage" and create a sub-section for "Wildcards". Then describe how those work with examples.